server/bin/gen_fake_report.py

```diff
@@ -38,16 +61,19 @@ class VirtWhoReportGenerator(AbstractVirtWhoReportGenerator):
     def _generate(self):
         report = {}
         for x in range(0, self.hypervisors):
-            report[str(uuid.uuid4())] = [self.gen_guest() for y in
-                range(0, self.guests)]
+            report[str(uuid.uuid4())] = [
+                self.gen_guest() for y in range(0, self.guests)
+            ]
         return report
 
 
 class AsyncVirtWhoReportGenerator(AbstractVirtWhoReportGenerator):
 
     def gen_hypervisor(self, num_guests, min_guests=0, max_guests=None):
-        return {"uuid": str(uuid.uuid4()),
-                "guests": [self.gen_guest() for x in range(0, num_guests)]}
+        return {"hypervisorId": {"hypervisorId": str(uuid.uuid4())},
```

Comment:
The `hypervisorId` can be of 3 different formats (based on the config) of either: uuid, hostname, hwuuid. If the property is not set, it defaults to uuid. So I think by default we should be setting this to the same uuid that the `dmi.system.uuid` fact value is.

Btw, it could be a nice feature to have a hypervisorId format cli flag, to specify which of the 3 formats it should use, although not that important (so feel free to skip).

Reply:
I was unaware of this, but I checked the hypervisor_check_in spec, and it has the same working. Made few more changes to accommodate both ```uuid``` & ```hostname``` as hypervisorID formats. 
- For ```uuid``` format ```hypervisorId``` & ```dmi.system.uuid``` fields will be same.
- For ```hostname``` format ```hypervisorId``` & ```name``` fields will be same 

The ```uuid``` format will remain as default.